package.json

```diff
@@ -31,8 +31,10 @@
     "lint-fix": "eslint tests/**/*.js tests/*.js src/**/*.js src/*.js --color --global requirejs --global require --fix",
     "test": "karma start --single-run --browsers ChromeHeadless",
     "test-drone": "karma start --single-run --browsers ChromeHeadlessNoSandbox",
+    "prepare": "npm run build:dev",
```

Comment:
Is that needed within the final PR in here? I'm aware of the local dev reasons for this - I'm wondering about sideeffects on the real build/publish - THX

Reply:
I hope we can keep it including the source map thing, especially if we're going to do more development on the SDK in the near future.

But I have the same worry like yours. I'll research this

Reply:
We could also skip webpack totally - I did add webpack to get transpiling in place.
With the knowledge of today: any consuming project can transpile any dependency to it's own needs.

the sdk can be shippes as pure sources ... which would also make debugging on the consuming project (e.g. phoenix) easier

Reply:
if we use ES6, would transpiling be needed for running the tests in the browser ?

Reply:
tests have anyhow their own setup/config - this is unrelated to the way we distribute the package